.dockerignore

```diff
@@ -0,0 +1,4 @@
+# ignore Markdown files
+*.md
+# ignore tests
+connaisseur/tests/
```

Comment:
If you want marginally faster builds, you can also add `img/` since that accounts for ~2/3 of the repo size

Reply:
think that is excluded anyways. Checking...

Reply:
For build size, it is, since its not used. But for speed during building, the whole context (the . for docker build . ) is copied to the docker daemon. Is local copy of few MB, so :man_shrugging: , but if we put a 1GB file in that folder you'll notice

Reply:
will however only increase the build speed, but good point!